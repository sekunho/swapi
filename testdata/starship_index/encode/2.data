{"count":36,"next":"https://swapi.dev/api/people/?format=json&page=3","previous":"https://swapi.dev/api/people/?format=json&page=1","results":[{"MGLT":"70","cargo_capacity":"70000","consumables":"1 month","cost_in_credits":"unknown","created":"2014-12-15T13:00:56.332Z","crew":"1","edited":"2014-12-20T21:23:49.897Z","films":["https://swapi.dev/api/films/2/","https://swapi.dev/api/films/5/"],"hyperdrive_rating":"3.0","length":"21.5","manufacturer":"\"Kuat Systems Engineering\"","max_atmosphering_speed":"1000km","model":"\"Firespray-31-class patrol and attack\"","name":"\"Slave 1\"","passengers":"6","pilots":["https://swapi.dev/api/people/22/"],"starship_class":"Patrol Craft","url":"https://swapi.dev/api/starships/21/"},{"MGLT":"50","cargo_capacity":"80000","consumables":"2 months","cost_in_credits":"240000","created":"2014-12-15T13:04:47.235Z","crew":"6","edited":"2014-12-20T21:23:49.9Z","films":["https://swapi.dev/api/films/2/","https://swapi.dev/api/films/3/"],"hyperdrive_rating":"1.0","length":"20.0","manufacturer":"\"Sienar Fleet Systems\"","max_atmosphering_speed":"850km","model":"\"Lambda-class T-4a shuttle\"","name":"\"Imperial shuttle\"","passengers":"20","pilots":["https://swapi.dev/api/people/1/","https://swapi.dev/api/people/13/","https://swapi.dev/api/people/14/"],"starship_class":"Armed Government Transport","url":"https://swapi.dev/api/starships/22/"},{"MGLT":"40","cargo_capacity":"6000000","consumables":"2 years","cost_in_credits":"8500000","created":"2014-12-15T13:06:30.813Z","crew":"854","edited":"2014-12-20T21:23:49.902Z","films":["https://swapi.dev/api/films/2/","https://swapi.dev/api/films/3/"],"hyperdrive_rating":"2.0","length":"300.0","manufacturer":"\"Kuat Drive Yards\"","max_atmosphering_speed":"800km","model":"\"EF76 Nebulon-B escort frigate\"","name":"\"EF76 Nebulon-B escort frigate\"","passengers":"75","pilots":[],"starship_class":"Escort Ship","url":"https://swapi.dev/api/starships/23/"},{"MGLT":"60","cargo_capacity":"unknown","consumables":"2 years","cost_in_credits":"104000000","created":"2014-12-18T10:54:57.804Z","crew":"5400","edited":"2014-12-20T21:23:49.904Z","films":["https://swapi.dev/api/films/3/"],"hyperdrive_rating":"1.0","length":"1200.0","manufacturer":"\"Mon Calamari shipyards\"","max_atmosphering_speed":"n/a","model":"\"MC80 Liberty type Star Cruiser\"","name":"\"Calamari Cruiser\"","passengers":"1200","pilots":[],"starship_class":"Star Cruiser","url":"https://swapi.dev/api/starships/27/"},{"MGLT":"120","cargo_capacity":"40","consumables":"1 week","cost_in_credits":"175000","created":"2014-12-18T11:16:34.542Z","crew":"1","edited":"2014-12-20T21:23:49.907Z","films":["https://swapi.dev/api/films/3/"],"hyperdrive_rating":"1.0","length":"9.6","manufacturer":"\"Alliance Underground Engineering, Incom Corporation\"","max_atmosphering_speed":"1300km","model":"\"RZ-1 A-wing Interceptor\"","name":"\"A-wing\"","passengers":"0","pilots":["https://swapi.dev/api/people/29/"],"starship_class":"Starfighter","url":"https://swapi.dev/api/starships/28/"},{"MGLT":"91","cargo_capacity":"45","consumables":"1 week","cost_in_credits":"220000","created":"2014-12-18T11:18:04.763Z","crew":"1","edited":"2014-12-20T21:23:49.909Z","films":["https://swapi.dev/api/films/3/"],"hyperdrive_rating":"2.0","length":"16.9","manufacturer":"\"Slayn & Korpil\"","max_atmosphering_speed":"950km","model":"\"A/SF-01 B-wing starfighter\"","name":"\"B-wing\"","passengers":"0","pilots":[],"starship_class":"Assault Starfighter","url":"https://swapi.dev/api/starships/29/"},{"MGLT":"unknown","cargo_capacity":"unknown","consumables":"unknown","cost_in_credits":"unknown","created":"2014-12-19T17:01:31.488Z","crew":"9","edited":"2014-12-20T21:23:49.912Z","films":["https://swapi.dev/api/films/4/"],"hyperdrive_rating":"2.0","length":"115.0","manufacturer":"\"Corellian Engineering Corporation\"","max_atmosphering_speed":"900km","model":"\"Consular-class cruiser\"","name":"\"Republic Cruiser\"","passengers":"16","pilots":[],"starship_class":"Space Cruiser","url":"https://swapi.dev/api/starships/31/"},{"MGLT":"unknown","cargo_capacity":"4000000000","consumables":"500 days","cost_in_credits":"unknown","created":"2014-12-19T17:04:06.323Z","crew":"175","edited":"2014-12-20T21:23:49.915Z","films":["https://swapi.dev/api/films/4/","https://swapi.dev/api/films/5/","https://swapi.dev/api/films/6/"],"hyperdrive_rating":"2.0","length":"3170.0","manufacturer":"\"Hoersch-Kessel Drive, Inc.\"","max_atmosphering_speed":"n/a","model":"\"Lucrehulk-class Droid Control Ship\"","name":"\"Droid control ship\"","passengers":"139000","pilots":[],"starship_class":"Droid Control Ship","url":"https://swapi.dev/api/starships/32/"},{"MGLT":"unknown","cargo_capacity":"65","consumables":"7 days","cost_in_credits":"200000","created":"2014-12-19T17:39:17.582Z","crew":"1","edited":"2014-12-20T21:23:49.917Z","films":["https://swapi.dev/api/films/4/","https://swapi.dev/api/films/5/"],"hyperdrive_rating":"1.0","length":"11.0","manufacturer":"\"Theed Palace Space Vessel Engineering Corps\"","max_atmosphering_speed":"1100km","model":"\"N-1 starfighter\"","name":"\"Naboo fighter\"","passengers":"0","pilots":["https://swapi.dev/api/people/11/","https://swapi.dev/api/people/35/","https://swapi.dev/api/people/60/"],"starship_class":"Starfighter","url":"https://swapi.dev/api/starships/39/"},{"MGLT":"unknown","cargo_capacity":"unknown","consumables":"unknown","cost_in_credits":"unknown","created":"2014-12-19T17:45:03.506Z","crew":"8","edited":"2014-12-20T21:23:49.919Z","films":["https://swapi.dev/api/films/4/"],"hyperdrive_rating":"1.8","length":"76.0","manufacturer":"\"Theed Palace Space Vessel Engineering Corps, Nubia Star Drives\"","max_atmosphering_speed":"920km","model":"\"J-type 327 Nubian royal starship\"","name":"\"Naboo Royal Starship\"","passengers":"unknown","pilots":["https://swapi.dev/api/people/39/"],"starship_class":"Yacht","url":"https://swapi.dev/api/starships/40/"}]}