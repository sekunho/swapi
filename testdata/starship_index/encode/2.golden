{"next":"https://swapi.dev/api/people/?format=json&page=3","results":[{"length":"21.5","starship_class":"Patrol Craft","MGLT":"70","films":["https://swapi.dev/api/films/2/","https://swapi.dev/api/films/5/"],"pilots":["https://swapi.dev/api/people/22/"],"url":"https://swapi.dev/api/starships/21/","passengers":"6","name":"\"Slave 1\"","model":"\"Firespray-31-class patrol and attack\"","edited":"2014-12-20T21:23:49.897Z","cargo_capacity":"70000","manufacturer":"\"Kuat Systems Engineering\"","cost_in_credits":"unknown","consumables":"1 month","created":"2014-12-15T13:00:56.332Z","hyperdrive_rating":"3.0","max_atmosphering_speed":"1000","crew":"1"},{"length":"20.0","starship_class":"Armed Government Transport","MGLT":"50","films":["https://swapi.dev/api/films/2/","https://swapi.dev/api/films/3/"],"pilots":["https://swapi.dev/api/people/1/","https://swapi.dev/api/people/13/","https://swapi.dev/api/people/14/"],"url":"https://swapi.dev/api/starships/22/","passengers":"20","name":"\"Imperial shuttle\"","model":"\"Lambda-class T-4a shuttle\"","edited":"2014-12-20T21:23:49.9Z","cargo_capacity":"80000","manufacturer":"\"Sienar Fleet Systems\"","cost_in_credits":"240000","consumables":"2 months","created":"2014-12-15T13:04:47.235Z","hyperdrive_rating":"1.0","max_atmosphering_speed":"850","crew":"6"},{"length":"300.0","starship_class":"Escort Ship","MGLT":"40","films":["https://swapi.dev/api/films/2/","https://swapi.dev/api/films/3/"],"pilots":[],"url":"https://swapi.dev/api/starships/23/","passengers":"75","name":"\"EF76 Nebulon-B escort frigate\"","model":"\"EF76 Nebulon-B escort frigate\"","edited":"2014-12-20T21:23:49.902Z","cargo_capacity":"6000000","manufacturer":"\"Kuat Drive Yards\"","cost_in_credits":"8500000","consumables":"2 years","created":"2014-12-15T13:06:30.813Z","hyperdrive_rating":"2.0","max_atmosphering_speed":"800","crew":"854"},{"length":"1200.0","starship_class":"Star Cruiser","MGLT":"60","films":["https://swapi.dev/api/films/3/"],"pilots":[],"url":"https://swapi.dev/api/starships/27/","passengers":"1200","name":"\"Calamari Cruiser\"","model":"\"MC80 Liberty type Star Cruiser\"","edited":"2014-12-20T21:23:49.904Z","cargo_capacity":"unknown","manufacturer":"\"Mon Calamari shipyards\"","cost_in_credits":"104000000","consumables":"2 years","created":"2014-12-18T10:54:57.804Z","hyperdrive_rating":"1.0","max_atmosphering_speed":"n/a","crew":"5400"},{"length":"9.6","starship_class":"Starfighter","MGLT":"120","films":["https://swapi.dev/api/films/3/"],"pilots":["https://swapi.dev/api/people/29/"],"url":"https://swapi.dev/api/starships/28/","passengers":"0","name":"\"A-wing\"","model":"\"RZ-1 A-wing Interceptor\"","edited":"2014-12-20T21:23:49.907Z","cargo_capacity":"40","manufacturer":"\"Alliance Underground Engineering, Incom Corporation\"","cost_in_credits":"175000","consumables":"1 week","created":"2014-12-18T11:16:34.542Z","hyperdrive_rating":"1.0","max_atmosphering_speed":"1300","crew":"1"},{"length":"16.9","starship_class":"Assault Starfighter","MGLT":"91","films":["https://swapi.dev/api/films/3/"],"pilots":[],"url":"https://swapi.dev/api/starships/29/","passengers":"0","name":"\"B-wing\"","model":"\"A/SF-01 B-wing starfighter\"","edited":"2014-12-20T21:23:49.909Z","cargo_capacity":"45","manufacturer":"\"Slayn & Korpil\"","cost_in_credits":"220000","consumables":"1 week","created":"2014-12-18T11:18:04.763Z","hyperdrive_rating":"2.0","max_atmosphering_speed":"950","crew":"1"},{"length":"115.0","starship_class":"Space Cruiser","MGLT":"unknown","films":["https://swapi.dev/api/films/4/"],"pilots":[],"url":"https://swapi.dev/api/starships/31/","passengers":"16","name":"\"Republic Cruiser\"","model":"\"Consular-class cruiser\"","edited":"2014-12-20T21:23:49.912Z","cargo_capacity":"unknown","manufacturer":"\"Corellian Engineering Corporation\"","cost_in_credits":"unknown","consumables":"unknown","created":"2014-12-19T17:01:31.488Z","hyperdrive_rating":"2.0","max_atmosphering_speed":"900","crew":"9"},{"length":"3170.0","starship_class":"Droid Control Ship","MGLT":"unknown","films":["https://swapi.dev/api/films/4/","https://swapi.dev/api/films/5/","https://swapi.dev/api/films/6/"],"pilots":[],"url":"https://swapi.dev/api/starships/32/","passengers":"139000","name":"\"Droid control ship\"","model":"\"Lucrehulk-class Droid Control Ship\"","edited":"2014-12-20T21:23:49.915Z","cargo_capacity":"4000000000","manufacturer":"\"Hoersch-Kessel Drive, Inc.\"","cost_in_credits":"unknown","consumables":"500 days","created":"2014-12-19T17:04:06.323Z","hyperdrive_rating":"2.0","max_atmosphering_speed":"n/a","crew":"175"},{"length":"11.0","starship_class":"Starfighter","MGLT":"unknown","films":["https://swapi.dev/api/films/4/","https://swapi.dev/api/films/5/"],"pilots":["https://swapi.dev/api/people/11/","https://swapi.dev/api/people/35/","https://swapi.dev/api/people/60/"],"url":"https://swapi.dev/api/starships/39/","passengers":"0","name":"\"Naboo fighter\"","model":"\"N-1 starfighter\"","edited":"2014-12-20T21:23:49.917Z","cargo_capacity":"65","manufacturer":"\"Theed Palace Space Vessel Engineering Corps\"","cost_in_credits":"200000","consumables":"7 days","created":"2014-12-19T17:39:17.582Z","hyperdrive_rating":"1.0","max_atmosphering_speed":"1100","crew":"1"},{"length":"76.0","starship_class":"Yacht","MGLT":"unknown","films":["https://swapi.dev/api/films/4/"],"pilots":["https://swapi.dev/api/people/39/"],"url":"https://swapi.dev/api/starships/40/","passengers":"unknown","name":"\"Naboo Royal Starship\"","model":"\"J-type 327 Nubian royal starship\"","edited":"2014-12-20T21:23:49.919Z","cargo_capacity":"unknown","manufacturer":"\"Theed Palace Space Vessel Engineering Corps, Nubia Star Drives\"","cost_in_credits":"unknown","consumables":"unknown","created":"2014-12-19T17:45:03.506Z","hyperdrive_rating":"1.8","max_atmosphering_speed":"920","crew":"8"}],"previous":"https://swapi.dev/api/people/?format=json&page=1","count":36}